src/cameras/src/pm/Acquisition.cpp

```diff
@@ -214,6 +214,12 @@ void pm::Acquisition<F, C>::frameWriterThread() {
                         m_latestFrame = frame;
                     }
 
+                    // make subdirectory to write to
+                    if (!std::filesystem::exists(filePath)) {
+                        std::filesystem::create_directories(filePath);
+                        spdlog::info("Acquisition being written under directory: {}", filePath.string());
+                    }
+
```

Comment:
I put this here so that the directories are only created if a TIFF file is being written, so that unnecessary directories aren't made if something else goes wrong